technic/machines/switching_station_globalstep.lua

```diff
@@ -111,34 +72,30 @@ minetest.register_globalstep(function(dtime)
 
 				-- overload detection
 				if switch_diff > 250000 then
-					switch.skip = 30
+					network.skip = 30
 				elseif switch_diff > 150000 then
-					switch.skip = 20
+					network.skip = 20
 				elseif switch_diff > 75000 then
-					switch.skip = 10
+					network.skip = 10
 				elseif switch_diff > 50000 then
-					switch.skip = 2
+					network.skip = 2
```

Comment:
It might be possible to divide laggy/large network execution into buckets where single cycle executes only part of network, there's few problems with this btu I think everything can be solved without complicating things too much:
* Network has 20 machines, 10 first gets executed, first few machines gets removed, next cycle is not balanced anymore without careful index tracking, things gets complicated.
* Supply converters might do weird things like free energy or cutting off continuously for short periods of time.
* Network output probably has to be cached, demand will fluctuate depending on what machines got selected for partial execution. Demand is significantly lower than what it is when whole network gets executed as whole. This might not be a problem as if it gets back to full cycles it will simply stop if demand > supply.
* Would be very good if battery arrays would be cached as single huge virtual battery unit and actual batteries updated during one separate cycle, destroying and forcing rebuild of network between cycles (constructor/piston/node breaker) if timing is correct might create infinite battery capacity and cause even more lag while giving benefits for user.

Probably better to let it be like that for now and see later if there's better solutions for that, changing it now might complicate things too much.